### ISSUE TITLE
error while fetching extensions. failed to fetch

### ISSUE THREAD

**Issue by marconaccini** [microsoft/vscode#233848]
Type: <b>Feature Request</b>

I got error "error while fetching extensions. failed to fetch" reading extensions

VS Code version: Code 1.95.2 (e8653663e8840adaf45af01eab5c627a5af81807, 2024-11-07T11:07:22.054Z)
OS version: Windows_NT x64 10.0.19045
Modes:

**Comment by IanMatthewHuff** [comment]
I'm seeing this same issue in current insiders.vscode.dev. Extension loading possibly being blocked by CORS?
![Image](https://github.com/user-attachments/assets/bbf02ea1-fd00-435c-9e51-851b26418587)

**Comment by kaifastromai** [comment]
I think the extension marketplace is down. Not sure if the same server/s is/are backing both services, but I get a timeout when trying to access https://marketplace.visualstudio.com/VSCode

**Comment by sandy081** [comment]
@IanMatthewHuff Extracted your comment to a new issue, it seems unrelated to this.

@marconaccini Can you please share following information if you are reproducing

- F1 > Developer: Toggle Developer Tools > Console
